drivers/gles3/texture_loader_gles3.cpp

```diff
@@ -51,7 +51,7 @@ RES ResourceFormatGLES2Texture::load(const String &p_path, const String &p_origi
 	uint8_t **row_p = memnew_arr(uint8_t *, height);
 
 	for (unsigned int i = 0; i < height; i++) {
-		row_p[i] = 0; //No colors any more, I want them to turn black
+		row_p[i] = nullptr; //No colors any more, I want them to turn black
```

Comment:
Given the comment, I wonder if that's not hiding a bug here. I think the intend is to set each `uint8_t` to 0, not their pointers to `nullptr`. CC @lawnjelly

Reply:
What is the point of this array, `row_p`? From what I see, it's just created, assigned, then deleted. I'm removing it for now.

Reply:
It's an array of pointers to some memory. It's used to modify that memory. So it definitely does something (or is meant to) and shouldn't be removed IMO.